manifests/base/runtime-rocm-pytorch-imagestream.yaml

```
@@ -18,12 +18,12 @@ spec:
         opendatahub.io/runtime-image-metadata: |
           [
             {
-              "display_name": "PyTorch with ROCm and Python 3.12 (UBI9)",
+              "display_name": "Runtime | Pytorch | ROCm| Python 3.12",
```

```suggestion
              "display_name": "Runtime | Pytorch | ROCm | Python 3.12",
```
this need to updated in several places